### PR TITLE
Cloudshell tutorial

### LINKED ISSUE #6: Create Cloudshell Tutorial
Let users try out gcsbeat with a quick tutorial and a cloudshell link from the main README.md

### PR DESCRIPTION
**Pull request by josephlewis42**
I've added a basic cloudshell tutorial about how to get going. Should fix #6 and provide a good source of info for people needing to debug.

### CODE REVIEW

File: tutorial/config.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  delete: false
+  file_matches: "*.log"
+  file_exclude: "bak_*"
+  metadata_key: x-goog-meta-gcsbeat-tutorial
+  codec: "text"
+  unpack_gzip: false
+
+output.console:
+  pretty: true
+
```
Comment (rosbo) [inline tutorial/config.yml]: nit: rm extra line

File: tutorial/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+Show the contents of the configuration file:
+
+	cat tutorial/config.conf
+
+## Run GCSBeat
+
+Execute `gcsbeat` to fetch the
+
+	./gcsbeat -e -v -c tutorial/config.conf
```
Comment (rosbo) [inline tutorial/tutorial.md]: Consider adding a line about what the user should expect to see.

File: tutorial/config.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  # The path to the key to authenticate your user to the bucket.
+  # this service user _should_ have the `storage.objects.update`
+  # permission so it can create metadata on the object preventing
+  # it from being scanned multiple times.
+  json_key_file: ${PWD}/gcsbeat-tutorial-key.json
+
+  delete: false
+  file_matches: "*.log"
+  file_exclude: "bak_*"
```
Comment (rosbo) [inline tutorial/config.yml]: your bak filename in the demo has a dash `-` not an underscore `_`.

**Review by rosbo — CHANGES_REQUESTED** [review]

**Comment by rosbo** [comment]
I like the CloudShell link. Super convenient.

**Comment by josephlewis42** [comment]
@rosbo Alright, I think I cleaned it all up. That's what I get for editing in my VDE and the CloudShell simultaneously :-)

**Review by rosbo — APPROVED** [review]

**Pull request merged**